csharp/src/Apache.Arrow/Apache.Arrow.csproj

```diff
@@ -7,18 +7,16 @@
     <Description>Apache Arrow is a cross-language development platform for in-memory data. It specifies a standardized language-independent columnar memory format for flat and hierarchical data, organized for efficient analytic operations on modern hardware.</Description>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(IsWindows)'=='true'">
+  <PropertyGroup>
```

Comment:
Just checking this was intentionally included in the PR? Seems reasonable to now build for net462 on non-Windows too, this built fine for me on Linux and I guess something made sure the reference assemblies / targeting pack was available.

Reply:
It was intentional, though arguably I should have put it into a separate PR. What I didn't realize when adding net462 support back was that the official NuGet packages aren't built on Windows and so they were missing the 4.6.2 assembly.